performance-tests/wrap-performance-test.ts

```diff
@@ -0,0 +1,96 @@
+import { Output, wrap } from "../src/implementations/wrap";
+import { performance } from "perf_hooks";
+import { existsSync, mkdirSync, rmdirSync, promises } from "fs";
+import { SchemaId } from "@govtechsg/open-attestation";
+import { join } from "path";
+
+const DEFAULT_NUMBER_OF_FILE = 2;
+const DEFAULT_ITERATION = 1;
+const DEFAULT_FILE_PATH = join(__dirname, "unwrapped_document.json");
+const INPUT_UNWRAPPED_FILE_FOLDER = join(__dirname, "setup", "raw-documents");
+const OUTPUT_WRAPPED_FILE_FOLDER = join(__dirname, "setup", "wrapped-documents");
+
+// Setup number of files
+const setup = async (filePath: string, numberOfFiles: number): Promise<void> => {
+  console.info("Setup up files for testing");
+  const fileNameArray = filePath.split("/").pop();
```

Comment:
doesn't work on windows =) https://nodejs.org/api/path.html#path_path_basename_path_ext

Reply:
Can get extension too https://nodejs.org/api/path.html#path_path_extname_path

Reply:
changing to this

const fileName = parse(filePath).name;
const fileExtension = parse(filePath).ext;